exporter/awsemfexporter/factory.go

```diff
@@ -47,18 +49,32 @@ func createDefaultConfig() component.Config {
 		LogStreamName:                   "",
 		Namespace:                       "",
 		DimensionRollupOption:           "ZeroAndSingleDimensionRollup",
+		EnableEMFVersion1:               true,
```

Comment:
The [default EMF version is v1](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/exporter/awsemfexporter/metric_translator.go#L374-L378) and the public document mentions [V1 more than V0](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Embedded_Metric_Format_Specification.html#CloudWatch_Embedded_Metric_Format_Specification_structure). Therefore, enable EMF version with default is true